Core/DISET/private/BaseClient.py

```diff
@@ -140,7 +140,7 @@ def __discoverCredentialsToUse( self ):
     if self.KW_PROXY_CHAIN in self.kwargs:
       try:
         self.kwargs[ self.KW_PROXY_STRING ] = self.kwargs[ self.KW_PROXY_CHAIN ].dumpAllToString()[ 'Value' ]
-        del( self.kwargs[ self.KW_PROXY_CHAIN ] )
+        del self.kwargs[ self.KW_PROXY_CHAIN ]
```

Comment:
I don't know. This is really deep, and also old code. I don't want to change anything in it atm.